### PR TITLE
 case insensitive checking

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
Shouldn't be case sensitive adding emails

# What has changed
makes it case insensitive

# How to test?
Try to add a user to a group, manually change value in text box to change the case of some letters.  Check that the email it adds still correct (in this case it will use the capitalization/case already stored for the email

# Links
[

](https://trello.com/c/IQ5b0zYl/3309-defect-adding-user-emails-in-support-tool-is-case-sensitive-5)

# Screenshots (if appropriate):

### CODE REVIEW

File: ui/src/AddUserToGroup.js [hunk 1]
```diff
@@ -68,17 +68,28 @@ function AddUserToGroup(props) {
...
+    if (
+      location.state.existingEmailsInGroup.filter(
+        (existingEmail) =>
+          existingEmail.toLowerCase() === newEmail.toLowerCase()
+      ).length > 0
+    ) {
+      return true;
+    }
+
+    return false;
```
Comment (liamtoozer) [inline ui/src/AddUserToGroup.js:92]:
```suggestion
    return location.state.existingEmailsInGroup.filter(
        (existingEmail) =>
            existingEmail.toLowerCase() === newEmail.toLowerCase()
    ).length > 0;
```

**Review by liamtoozer — APPROVED** [review]
LGTM 👍  Tested locally

**Review by ryangrundy7 — APPROVED** [review]

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
I think the backend also needs updating, the user look-ups there must also be case-insensitive.

**Review by AdamHawtin — APPROVED** [review]
LGTM

**Pull request merged**